### PR TITLE
Update moment map test so CI passes

### PR DESCRIPTION
**Pull request by rosteen**
All our CI failing is too annoying to wait for comment from SMEs on whether this merits an upstream change/fix as well. We can update this again if needed later.

### CODE REVIEW

**Comment by rosteen** [comment]
Apparently the value at the mouseover location changed as well, which worries me. I don't have time to think about it further right this moment though.

File: jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py [hunk 1]
```diff
@@ -47,9 +47,9 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
     label_mouseover = cubeviz_helper.app.session.application._tools['g-coords-info']
     label_mouseover._viewer_mouse_event(flux_viewer, {'event': 'mousemove',
                                                       'domain': {'x': 0, 'y': 0}})
-    assert flux_viewer.state.slices == (0, 0, 1)
+    assert flux_viewer.state.slices == (0, 0, 0)
```
Comment (pllim) [inline jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py:50]: This is cause by glue-core v1.9.0 . I tried to bisect glue-core and git blamed https://github.com/glue-viz/glue/commit/da68d1a63adfd6f91aac1e7d2cf864a359c33baa though my bisect-fu isn't strong so we need @astrofrog to explain how that commit could have changed the `viewer.state.slices` default behavior here.

Comment (pllim) [reply jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py:50]: Maybe we should go with #2136 instead if this behavior change is undesirable.

**Comment by pllim** [comment]
I dug upstream and maybe we can ask @astrofrog about these findings:

* https://github.com/glue-viz/glue/commit/da68d1a63adfd6f91aac1e7d2cf864a359c33baa#r107847297
* https://github.com/glue-viz/glue-jupyter/pull/349

**Comment by rosteen** [comment]
Closing in favor of #2136 .

**Pull request closed without merging**